azure-client-runtime/src/main/java/com/microsoft/azure/AzureAsyncOperationPollStrategy.java

```diff
@@ -73,37 +75,50 @@ else if (pollingSucceeded) {
 
     @Override
     public Single<HttpResponse> updateFromAsync(final HttpResponse httpPollResponse) {
+        if (400 <= httpPollResponse.statusCode()) {
```

Comment:
Not a big thing but I encourage putting constants on the right side of a comparison

Reply:
Done.